tests/threshold/detection_filter-rule-hostsrc/test.rules

```diff
@@ -0,0 +1,6 @@
+alert icmp any any -> any any (itype:8; sid:1;)
+alert icmp any any -> any any (itype:8; detection_filter:track by_src, count 1, seconds 3600; sid:2;)
+alert icmp any any -> any any (itype:8; detection_filter:track by_src, count 2, seconds 3600; sid:3;)
+alert icmp any any -> any any (itype:8; detection_filter:track by_src, count 4, seconds 3600; sid:4;)
+alert icmp any any -> any any (itype:8; detection_filter:track by_src, count 7, seconds 3600; sid:5;)
+alert icmp any any -> any any (itype:8; detection_filter:track by_src, count 8, seconds 3600; sid:6;)
```

Comment:
Why this test ?
As there is only one flow, we do not see the difference between ` detection_filter:track by_src` and ` detection_filter:track by_flow` done in tests/threshold/detection_filter-rule-flow

Reply:
it tests a different code path

Reply:
Yes, but we do not prove that since we get the same behavior